packages/react-core/src/components/Wizard/__tests__/Wizard.test.tsx

```diff
@@ -205,3 +205,50 @@ describe('Wizard', () => {
     expect(main).toHaveAttribute('aria-labelledby', 'main-aria-labelledby');
   });
 });
+
+  test.only('wiz with disabled steps', () => {
+    const steps: WizardStep[] = [
+      { name: 'A', component: <p>Step 1</p>},
+      { name: 'B', component: <p>Step 2</p>, isDisabled: true },
+      { name: 'C', component: <p>Step 3</p>},
+      { name: 'E', component: <p>Step 4</p>, isDisabled: true },
+      { name: 'G', component: <p>Step 5</p> },
+
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+    const button1Disabled = screen.getByRole('button',{ name: "B" });
+    const button2Disabled = screen.getByRole('button',{ name: "E" });
+
+    expect(button1Disabled).toHaveAttribute('disabled');
```

Comment:
Rather than using `toHaveAttribute('disabled')` I would like to see the `toBeDisabled()` matcher from jest-dom used for these tests.